grpo-finetuning-qwen3/agent-skill/grpo-finetune/agent_demo.py

```
@@ -0,0 +1,141 @@
+
+            for field in REQUIRED_FIELDS:
+                val = result.get(field, "—")
+                print(f"  {GRAY}{field:<10}{RESET}  {val}")
+
+            print()
+            if valid:
+                print(f"  {GREEN}✓  Schema valid{RESET}    {GRAY}{elapsed}s{RESET}")
+            else:
+                print(f"  {RED}✗  Schema mismatch{RESET}  {GRAY}{elapsed}s{RESET}")
+
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)